src/shared/lib/getCanonicalMutationType.ts

```diff
@@ -98,6 +98,7 @@ export function getProteinImpactTypeFromCanonical(mutationType:CanonicalMutation
         case "fusion":
         case "silent":
```

Comment:
that makes sense from a code simplicity view but I think it's better in this case to spell it out so we can see exactly what's getting assigned to other